daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -776,6 +783,7 @@ final case class Compiler(packages: PackageId PartialFunction Package) {
         Map.empty,
         0,
         withEnv { _ =>
+          env = env.incrPos // byKey flag
```

Comment:
I think `byKey` still needs to be reflected in line 777.

Reply:
Indeed. Line 775 as well.